scout/server/blueprints/institutes/templates/overview/causatives.html

```diff
@@ -28,7 +29,7 @@
 <div class="container-float">
   <div class="row" id="body-row"> <!--sidebar and main container are on the same row-->
     <div class="col">
-        {{ causatives() }}
+        {{ variant_list_content(institute, causatives, acmg_map, callers) }}
```

Comment:
There are slightly different styles active on the verified and causatives pages; not really that important, but currently the visited links on Variants follows the even-odd scheme with links not turning purple after visiting, whereas Causatives have `visited:--bs-purple`. Both are ok, but I don't think the visited link color serves much purpose here really.

Reply:
Link color that changes is not so bad, especially if you have more than one variant in the same gene. I'd keep it as it is.